Dockerfile

```diff
@@ -32,6 +32,9 @@ RUN sed 's/.ALL:ALL./(ALL) NOPASSWD:/' -i /etc/sudoers
 RUN wget "http://ftp.halifax.rwth-aachen.de/eclipse//technology/epp/downloads/release/2019-03/R/eclipse-jee-2019-03-R-linux-gtk-x86_64.tar.gz" -O /tmp/eclipse.tar.gz;\
     cd /opt ; tar xzf /tmp/eclipse.tar.gz;\
     rm /tmp/eclipse.tar.gz
+RUN wget https://projectlombok.org/downloads/lombok.jar /tmp/lombok.jar;\
+    /tmp/lombok.jar install /opt/eclipse;\
```

Comment:
java -jar is missing

Reply:
or chmod +x /tmp/lombok.jar. I didn't have enough free space to try it :(

Reply:
chmod +x  would rely on some linux-only binary format support, which is turned off in some cases for security reasons.